### ISSUE TITLE
Looks like this is causing Interpreter to crash when running at least vulkaninfo (without thunks).

### ISSUE THREAD

**Issue by Sonicadvance1** [FEX-Emu/FEX#1793]
Looks like this is causing Interpreter to crash when running at least vulkaninfo (without thunks).

Couple of different crashes.
```
Thread 1 "FEXLoader" received signal SIGSEGV, Segmentation fault.
Xbyak::CodeArray::isAutoGrow (this=0x0) at /mnt/Work/Work/work/FEXNew/External/xbyak/xbyak/xbyak.h:1107
1107            bool isAutoGrow() const { return type_ == AUTO_GROW; }
(gdb) bt
#0  Xbyak::CodeArray::isAutoGrow (this=0x0) at /mnt/Work/Work/work/FEXNew/External/xbyak/xbyak/xbyak.h:1107
#1  0x00005555557f4cb7 in Xbyak::LabelManager::define_inner<std::unordered_map<int, Xbyak::LabelManager::ClabelVal, std::hash<int>, std::equal_to<int>, std::allocator<std::pair<int const, Xbyak::LabelManager::ClabelVal> > >, std::unordered_multimap<int, Xbyak::JmpLabel const, std::hash<int>, std::equal_to<int>, std::allocator<std::pair<int const, Xbyak::JmpLabel const> > >, int> (this=0x7ffff7763140, defList=std::unordered_map with 0 elements, undefList=std::unordered_multimap with 0 elements, labelId=@0x7fffffffc6ac: 1, addrOffset=24) at /mnt/Work/Work/work/FEXNew/External/xbyak/xbyak/xbyak.h:1345
#2  0x00005555557f49e0 in Xbyak::LabelManager::defineClabel (this=0x7ffff7763140, label=...) at /mnt/Work/Work/work/FEXNew/External/xbyak/xbyak/xbyak.h:1451
#3  0x00005555557e80c4 in Xbyak::CodeGenerator::L (this=0x7ffff77630b0, label=...) at /mnt/Work/Work/work/FEXNew/External/xbyak/xbyak/xbyak.h:2370
#4  0x00005555557e8f98 in FEXCore::CPU::X86Dispatcher::GenerateInterpreterTrampoline (this=0x7ffff7763000, CodeBuffer=0x7fffe273a9d8 "L\211\367H\215\065") at /mnt/Work/Work/work/FEXNew/External/FEXCore/Source/Interface/Core/Dispatcher/X86Dispatcher.cpp:456
#5  0x0000555555897853 in FEXCore::CPU::InterpreterCore::CompileCode (this=0x7ffff762d480, Entry=140736973056609, IR=0x7ffff7795d30, DebugData=0x7ffff7795d60, RAData=0x0, GDBEnabled=false) at /mnt/Work/Work/work/FEXNew/External/FEXCore/Source/Interface/Core/Interpreter/InterpreterCore.cpp:80
#6  0x00005555556f0074 in FEXCore::Context::Context::CompileCode (this=0x7ffff7694000, Thread=0x7ffff7690000, GuestRIP=140736973056609) at /mnt/Work/Work/work/FEXNew/External/FEXCore/Source/Interface/Core/Core.cpp:948
#7  0x00005555556f028f in FEXCore::Context::Context::CompileBlock (this=0x7ffff7694000, Frame=0x7ffff76901b0, GuestRIP=140736973056609) at /mnt/Work/Work/work/FEXNew/External/FEXCore/Source/Interface/Core/Core.cpp:989
#8  0x00005555556f0149 in FEXCore::Context::Context::CompileBlockJit (this=0x7ffff7694000, Frame=0x7ffff76901b0, GuestRIP=140736973056609) at /mnt/Work/Work/work/FEXNew/External/FEXCore/Source/Interface/Core/Core.cpp:959
```

or
```
terminate called after throwing an instance of 'Xbyak::Error'
  what():  label is not found
  ```
```
(gdb) disas $pc,+32
Dump of assembler code from 0x7fffd63ef8b2 to 0x7fffd63ef8d2:
=> 0x00007fffd63ef8b2:  add    BYTE PTR [rax],al
   0x00007fffd63ef8b4:  add    BYTE PTR [rax],al
   0x00007fffd63ef8b6:  add    BYTE PTR [rax],al
   0x00007fffd63ef8b8:  add    BYTE PTR [rax],al
   0x00007fffd63ef8ba:  add    BYTE PTR [rax],al
   ```

This is on an x86 host, interpreter, 500 inst, no-multiblock, Ubuntu 22.04 without rootfs.

_Originally posted by @Sonicadvance1 in https://github.com/FEX-Emu/FEX/issues/1782#issuecomment-1164140882_

**Comment by Sonicadvance1** [comment]
Looks like this holds true on AArch64 as well.
```
UNREACHABLE in /mnt/Work/Work/work/FEXNew/External/vixl/src/aarch64/instructions-aarch64.cc, line 989

Thread 1 "FEXInterpreter" received signal SIGABRT, Aborted.
__pthread_kill_implementation (threadid=549621534752, signo=signo@entry=6, no_tid=no_tid@entry=0) at ./nptl/pthread_kill.c:44
44      ./nptl/pthread_kill.c: No such file or directory.
(gdb) bt
#0  __pthread_kill_implementation (threadid=549621534752, signo=signo@entry=6, no_tid=no_tid@entry=0) at ./nptl/pthread_kill.c:44
#1  0x0000007ff7b6f254 in __pthread_kill_internal (signo=6, threadid=<optimised out>) at ./nptl/pthread_kill.c:78
#2  0x0000007ff7b2a67c in __GI_raise (sig=sig@entry=6) at ../sysdeps/posix/raise.c:26
#3  0x0000007ff7b17130 in __GI_abort () at ./stdlib/abort.c:79
#4  0x0000005555c73184 in vixl::aarch64::Instruction::SetBranchImmTarget (this=0x7fccb1adbc, target=0x7fccb1adc4)
    at /mnt/Work/Work/work/FEXNew/External/vixl/src/aarch64/instructions-aarch64.cc:989
#5  0x0000005555c72f38 in vixl::aarch64::Instruction::SetImmPCOffsetTarget (this=0x7fccb1adbc, target=0x7fccb1adc4)
    at /mnt/Work/Work/work/FEXNew/External/vixl/src/aarch64/instructions-aarch64.cc:941
#6  0x0000005555c12b58 in vixl::aarch64::Assembler::BindToOffset (this=0x7ff766d0b0, label=0x7fffffdc68, offset=12)
    at /mnt/Work/Work/work/FEXNew/External/vixl/src/aarch64/assembler-aarch64.cc:69
#7  0x0000005555c129c4 in vixl::aarch64::Assembler::bind (this=0x7ff766d0b0, label=0x7fffffdc68) at /mnt/Work/Work/work/FEXNew/External/vixl/src/aarch64/assembler-aarch64.cc:56
#8  0x000000555584a298 in FEXCore::CPU::Arm64Dispatcher::GenerateInterpreterTrampoline (this=0x7ff766d000, CodeBuffer=0x7fe23f7bf0 "\340\003\034\252\001")
    at /mnt/Work/Work/work/FEXNew/External/FEXCore/Source/Interface/Core/Dispatcher/Arm64Dispatcher.cpp:553
#9  0x00000055558fc814 in FEXCore::CPU::InterpreterCore::CompileCode (this=0x7ff762bc20, Entry=549616087667, IR=0x7ff76669c0, DebugData=0x7ff76669f0, RAData=0x0, GDBEnabled=false)
    at /mnt/Work/Work/work/FEXNew/External/FEXCore/Source/Interface/Core/Interpreter/InterpreterCore.cpp:80
#10 0x000000555574c400 in FEXCore::Context::Context::CompileCode (this=0x7ff768e000, Thread=0x7ff76ef800, GuestRIP=549616087667)
    at /mnt/Work/Work/work/FEXNew/External/FEXCore/Source/Interface/Core/Core.cpp:948
#11 0x000000555574c610 in FEXCore::Context::Context::CompileBlock (this=0x7ff768e000, Frame=0x7ff76ef9b0, GuestRIP=549616087667)
    at /mnt/Work/Work/work/FEXNew/External/FEXCore/Source/Interface/Core/Core.cpp:989
#12 0x000000555574c4fc in FEXCore::Context::Context::CompileBlockJit (this=0x7ff768e000, Frame=0x7ff76ef9b0, GuestRIP=549616087667)
    at /mnt/Work/Work/work/FEXNew/External/FEXCore/Source/Interface/Core/Core.cpp:959
#13 0x0000007ff7fed154 in ?? ()
#14 0x00000000f768e3d0 in ?? ()
```

**Comment by skmp** [comment]
Looking into it

**Comment by skmp** [comment]
Resolved with #1794
